config/v1alpha1/types_image_policy.go

```diff
@@ -143,6 +151,39 @@ type PolicyFulcioSubject struct {
 	SignedEmail string `json:"signedEmail"`
 }
 
+// PKI defines the root of trust based on Root CA(s) and corresponding intermediate certificates.
+// +openshift:enable:FeatureGate=SigstoreImageVerificationPKI
+type PKI struct {
+	// caRootsData contains base64-encoded data of a certificate bundle PEM file, which contains one or more CA roots in the PEM format.
+	// +kubebuilder:validation:Required
+	// +kubebuilder:validation:MaxLength=8192
+	CertificateAuthorityRootsData []byte `json:"caRootsData"`
```

Comment:
Since this is supposed to be in PEM format, and PEM has a predefined set of known characters, can we validate that this matches a particular regex of known characters please

Reply:
I think []byte already requires valid base64-encoded data. we can probably add validation to ensure this field starts with the base64 encoding of -----BEGIN CERTIFICATE-----. For the suffix, I did not find that the base64 data always ends with the base64 encoding of -----END CERTIFICATE-----. :thinking: